routers/productsRouter.js

```diff
@@ -1,10 +1,10 @@
-import controllers from '../controllers/index.js'
 import { Router } from 'express'
+import { productController } from '../controllers/index.js'
 
 const productsRouter = Router()
 
-productsRouter.get('/', controllers.getProducts)
-productsRouter.get('/category', controllers.getProduct)
-productsRouter.get('/:productId', controllers.getProductById)
+productsRouter.get('/', productController.getProducts)
```

Comment:
```suggestion
productsRouter.get('', productController.getProducts)
```
REST architecture의 여러 원칙중 안티패턴을 줄이고 REST를 사용하는 개발자들이 통일된 포맷의 엔드포인트를 작성할 수 있도록 정의한 uniform interface 원칙에 의하면 엔드포인트 끝에는 `/`를 사용하는 것을 지양하도록 하고 있습니다. 

이러한 부분도 크리티컬하진 않지만 REST를 배우고 이해하며 적용한 프로젝트이기에 
이에 따라 수정하면 좋을 것 같습니다~!

cartsRouter.js 같은 모듈에도 적용될 수 있는 부분을 찾아 수정부탁드리겠습니다. :)